### PR TITLE
Add More Test

### PR DESCRIPTION
**Pull request by tchiong**
Added AddAppCommand and AddAppCommandParser Tests

### CODE REVIEW

File: src/test/java/seedu/placebook/logic/parser/AddAppCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        assertParseSuccess(parser,
+                " id/1 a/vivocity start/01-01-2021 1000 end/01-01-2021 1200 ds/Halloween Sales",
+                new AddAppCommand(indexes, new Address("vivocity"),
+                        LocalDateTime.of(2021, 1, 1, 10, 0),
+                        LocalDateTime.of(2021, 1, 1, 12, 0),
+                        "Halloween Sales"));
+    }
+
+    @Test
+    public void parse_optionalFieldsMissing_success() {
```
Comment (nwhloo99) [inline src/test/java/seedu/placebook/logic/parser/AddAppCommandParserTest.java:32]: Technically description isnt an optionalField

**Review by nwhloo99 — COMMENTED** [review]
Maybe can add a multiple indexes test for the AddAppCommandParserTesst

File: src/test/java/seedu/placebook/logic/parser/AddAppCommandParserTest.java (hunk 1 above)
Comment (tchiong) [reply src/test/java/seedu/placebook/logic/parser/AddAppCommandParserTest.java:32]: Updated the Tests based on feedback!

**Review by nwhloo99 — APPROVED** [review]
LGTM

**Pull request merged**